src/components/layout/MainNav.astro

```diff
@@ -11,14 +11,9 @@ const navLinks = [
 ---
 
 <div
-  class="to-black-20 sticky top-0 z-10 h-20 w-full border-b border-white/5 bg-gradient-to-r from-black/40 via-black/25 mix-blend-screen backdrop-blur md:h-16"
+  class="to-black-10/10 via-black-20/10 sticky top-0 z-10 h-20 w-full border-b border-white/5 bg-gradient-to-r from-black/10 backdrop-blur md:h-16"
 >
-  <nav
-    class:list={[
-      styles.layout.container,
-      "flex flex-wrap md:flex-nowrap items-center gap-y-2 py-2 md:min-h-16 md:py-4",
-    ]}
-  >
+  <nav class:list={["flex flex-wrap md:flex-nowrap items-center gap-y-2 py-2 md:min-h-16 md:py-4 max-w-screen-xl"]}>
```

Comment:
```suggestion
  <nav class="flex flex-wrap md:flex-nowrap items-center gap-y-2 py-2 md:min-h-16 md:py-4 max-w-screen-xl">
```

If we go for this approach, I guess it can be simplified like this.